homeassistant/components/command_line/strings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "issues": {
+    "deprecated_platform_yaml": {
+      "title": "Command Line YAML configuration has been deprecated",
+      "description": "Configuring Command Line `{platform}` using YAML has been deprecated.\n\nConsult the documentation to move your YAML configuration to integration key and restart Home Assistant to resolve this issue."
```

Comment:
The first sentence sounds as if we deprecate YAML (for something else) while we're actually only requiring a different YAML structure.